.github/workflows/pr-merge.yml

```diff
@@ -0,0 +1,37 @@
+# This is a basic workflow to help you get started with Actions
+
+name: BZ Merge
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  pull_request_target:
+    branches:
+    - "*"
+    types:
+    - closed
+    
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in pnamearallel
+jobs:
+  # This workflow contains a single job called "build"
+  bz-on-pr-merge:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      - uses: docker://quay.io/konveyor/pr-merge-github-action
+        name: update bugzilla to modified
+        with:
+          bugzilla_token: ${{ secrets.BUGZILLA_TOKEN }}
+          org_repo: ${{ github.repository }}
+          pr_number: ${{ github.event.pull_request.number }}
+          bz_product: "Migration Toolkit for Containers"
+          title: ${{ github.event.pull_request.title }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          branch_to_release: "release-1.4.0:1.4.0,master:1.5.0"
```

Comment:
I see a release-1.3.2 branch but it is not in the mapping

Reply:
I removed it because we never actually had a 1.3.2 for this repo.